src/stores/closetStore.ts

```
@@ -0,0 +1,40 @@
+import type { ClosetItem } from "@/screens/Dressroom/dressroom.mock";
+import { MOCK_CLOSETS } from "@/screens/Dressroom/dressroom.mock";
+
+let closets: ClosetItem[] = [...MOCK_CLOSETS];
+
+const listeners = new Set<() => void>();
+
+function notify() {
+  listeners.forEach((cb) => cb());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)